Tests/UploadTests.swift

```diff
@@ -518,6 +518,32 @@ class UploadMultipartFormDataTestCase: BaseTestCase {
             XCTFail("Content-Type header value should not be nil")
         }
     }
+    
+    func testThatUploadingMultipartFormDataWithNotExistFileShouldFail() {
+        // Given
+        let urlString = "https://httpbin.org/post"
+        let imageURL = URL(string: "file://not_exists_file.jpg")!
```

Comment:
```suggestion
        let imageURL = URL(fileURLWithPath: "does_not_exist.jpg")
```

This makes it a bit safer.